mongo_scripts/create_indexes.js

```diff
@@ -0,0 +1,197 @@
+// Create indexes on all collections
```

Comment:
These files should go in the /resources directory.

Reply:
Moved to the resources directory